crates/edr_op/tests/integration/mod.rs

```diff
@@ -1,16 +1,30 @@
+#![cfg(feature = "test-remote")]
```

Comment:
Not all integration tests are guaranteed to be remote, so this should remain at a more granular level